### DISCUSSION TITLE
使用GO-CQHTTP 版（与 Mirai 二选一）一键 部署在 VPS   （centos)上，一直验证不成功

### DISCUSSION THREAD

**Discussion by hydpro** [lss233/chatgpt-mirai-qq-bot#1094] [General]
这是什么情况，要怎么处理，一直循环，登陆一直不成功
hatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:02] [INFO]: 按 Enter 继续....
chatgpt-qq-gocqhttp-1    |
chatgpt-qq-gocqhttp-1 exited with code 0
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:02] [INFO]: 当前版本:v1.1.0
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:02] [INFO]: 将使用 device.json 内的设备信息运行Bot.
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:03] [WARNING]: 警告: 未配置签名服务器, 这可能会导致登录 45 错误码或发  送消息被风控
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:03] [INFO]: Bot将在5秒后登录并开始信息处理, 按 Ctrl+C 取消.
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:08] [INFO]: 开始尝试登录并同步消息...
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:08] [INFO]: 使用协议: Android Pad 8.9.63.11390
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:08] [INFO]: 正在检查协议更新...
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:09] [INFO]: Protocol -> connect to server: 43.154.240.13:8080
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:09] [WARNING]: 登录需要滑条验证码, 请验证后重试.
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:09] [WARNING]: 请选择提交滑块ticket方式:
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:09] [WARNING]: 1. 自动提交
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:09] [WARNING]: 2. 手动抓取提交
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:09] [WARNING]: 请输入(1 - 2)：
chatgpt-qq-gocqhttp-1    | [2023-07-29 00:08:09] [WARNING]: 请前往该地址验证 -> https://captcha.go-cqhttp.org/captc  ha?id=YsFO3w7X&style=simple&aid=16&uin=20895099。。。。。。。。

**Comment by teddy-2** [comment]
点1，然后复制打开验证，然后等会再重开就行

**Reply by hydpro** [reply]
chatgpt-qq-gocqhttp-1 | [2023-07-29 00:08:09] [WARNING]: 请输入(1 - 2)：
chatgpt-qq-gocqhttp-1 | [2023-07-29 00:08:09] [WARNING]: 请前往该地址验证 -> https://captcha.go-cqhttp.org/captc ha?id=YsFO3w7X&style=simple&aid=16&uin=20895099。。。。。。。。  这个环节，根本不让输入1-2，直接就跳到验证，也可以在网页划动，一直验证不成功

**Reply by teddy-2** [reply]
或者去群里找个滑块工具啥的？要不就去群里问问

**Reply by hydpro** [reply]
滑块工具有显示，在页面中滑块提会验证成功，但日志一直提示失败，又验证。是用脚本一键生成的，不用做其它的配置吧

**Comment by qrzbing** [comment]
建议在国内服务器上登录

**Comment by 2558589594** [comment]
你这是使用docker的方式部署的吧，docker无法输入命令到那个终端，建议使用扫码，或者你把那个容器停了，单独在外面运行一个gocq
